test/e2e/federated-service.go

```diff
@@ -44,7 +44,7 @@ const (
 	KubeAPIQPS   = 20.0
 	KubeAPIBurst = 30
 
-	FederatedServiceTimeout = 5 * time.Minute
+	FederatedServiceTimeout = 30 * time.Second
```

Comment:
I would say keep it a minute?
Reducing time for a failing test is good, but more important is not to fail just because we didnt wait enough.